library/core/src/num/nonzero.rs

```diff
@@ -2265,3 +2265,33 @@ mod verify {
     nonzero_check!(u128, core::num::NonZeroU128, nonzero_check_new_unchecked_for_u128);
     nonzero_check!(usize, core::num::NonZeroUsize, nonzero_check_new_unchecked_for_usize);
 }
+
+#[cfg(kani)]
+mod macro_nonzero_check_rotate_left_and_right {
```

Comment:
What Caroly said: merge this into the `verify` module just above.

Reply:
Done! All in mod verify now